lib/interface_quda.cpp

```diff
@@ -1842,6 +1872,457 @@ void dslashQuda(void *h_out, void *h_in, QudaInvertParam *inv_param, QudaParity
   popVerbosity();
 }
 
+void shiftQuda(void *h_out, void *h_in, int dir, int sym, QudaInvertParam inv_param)
+{
+  profileCovDev.TPSTART(QUDA_PROFILE_TOTAL);
```

Comment:
The use of `profileCovDev` here should be cleaned up to be consistent with the "auto-profiling" we now have.  I can do this if it's easier.

Reply:
it's fixed in 32b92209db